package.json

```diff
@@ -32,9 +32,9 @@
     "browserify": "~7.0.0",
     "canvas": "~1.2.1",
     "leaflet": "1.0.3",
-    "mapnik": "cartodb/node-mapnik#1.4.15-cdb8",
```

Comment:
aren't we using our fork anymore?

/cc @dgaubert

Reply:
I would try to upgrade to version `3.5.14` from upstream.